pkg/yang/entry.go

```diff
@@ -1477,10 +1478,10 @@ func (s sortedErrors) Less(i, j int) bool {
 		case len(fj) < x && len(fi) < x:
 			return 0
 		}
-		return nless(fi[x], fj[x])
+		return nless(fi[x-1], fj[x-1])
```

Comment:
Can you add a comment here? Something like `// Both slices have this index: compare them.`